### PR TITLE
Clean up unused filter audio language

### PR DESCRIPTION
**Pull request by grafixeyehero**

### CODE REVIEW

**Comment by dkanada** [comment]
@taconaut I haven't seen this feature working before, has it been working for you?

File: src/components/filterdialog/filterdialog.js [hunk 1]
```diff
@@ -71,7 +69,7 @@ define(["dialogHelper", "globalize", "connectionManager", "events", "browser", "
...
+        "livetvchannels" != options.mode && "albums" != options.mode && "artists" != options.mode && "albumartists" != options.mode && "songs" != options.mode || hideByClass(context, "videoStandard"), enableDynamicFilters(options.mode) && (context.querySelector(".genreFilters").classList.remove("hide"), context.querySelector(".officialRatingFilters").classList.remove("hide"), context.querySelector(".tagFilters").classList.remove("hide"), context.querySelector(".yearFilters").classList.remove("hide")), "movies" != options.mode && "episodes" != options.mode || context.querySelector(".videoTypeFilters").classList.remove("hide"), options.mode, "movies" != options.mode && "series" != options.mode && "episodes" != options.mode || context.querySelector(".features").classList.remove("hide"), "series" == options.mode && context.querySelector(".seriesStatus").classList.remove("hide"), "episodes" == options.mode && showByClass(context, "episodeFilter")
```
[the remark this answers is not in the document]
Comment (grafixeyehero) [reply src/components/filterdialog/filterdialog.js]: done.

**Comment by grafixeyehero** [comment]
Before 
Screenshot (18)

After
Screenshot_2019-09-13 Jellyfin(1)

**Comment by JustAMan** [comment]
I don't like throwing out work that might be useful if finished, can we re-org the filter part (which I like basing on the screenshot) but retain both the functionality and the translations of half-implemented feature?

**Comment by grafixeyehero** [comment]
@JustAMan so you say keep audio language filter?

**Comment by dkanada** [comment]
@JustAMan I'd say we should just remove it. No one seems keen on implementing it and right now it's just a broken filter in the interface.

**Comment by joshuaboniface** [comment]
Agree with @dkanada here @JustAMan, we have the history in Git/GitHub, and can go back to it if someone does eventually implement it.

**Pull request merged**